src/components/CookieNotice/CookieNotice.scss

```diff
@@ -0,0 +1,59 @@
+@import "constants/style.scss";
+
+$cookie-consent-decline-button: #dadada;
+$cookie-consent-decline-text: #4d4d4d;
+$cookie-consent-button-text-size: 18px;
+
+.cookie-notice {
+  background: $color-light-gray;
+  color: $color-black;
+  position: fixed;
+  bottom: 0;
+  padding: $padding--small;
+}
+
+.cookie-notice__buttons {
+  display: flex;
+}
+
+.cookie-notice__button-accept {
+  background: $color-backlog-blue;
+  color: $color-white;
+  margin: 0 0 $margin--default $margin--default;
+  border: none;
+  padding: 5px 15px;
+  border-radius: 5px;
+  font-family: inherit;
```

Comment:
Don't think this does anything because inherit only inherits from the parent, not from greater ancestors